backend/segmentation_service.py

```diff
@@ -101,7 +102,7 @@ def process_ndpi_segmentation_request(
 
     # Get contours from the array
     saved_contours = get_saved_contours(level_array)
-    
+    print(f"Found {len(saved_contours)} contours")
```

Comment:
Stay consistent: use loggin, not printing.
```suggestion
    logging.info(f"Found {len(saved_contours)} contours")
```

Reply:
Changed.